nullaway/src/main/java/com/uber/nullaway/fixserialization/SeparatedValueDisplay.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.nullaway.fixserialization;
+
+/** Provides a method for displaying object information as string. */
+public interface SeparatedValueDisplay {
```

Comment:
If we specialize to CSV can we get rid of this interface?

Reply:
Hi @msridhar, actually I was inspired by this name and we are not actually separating values with ```,``` as in ```CSV```. That is why we have the ```delimiter``` as a parameter. The problem with ```,``` is that method signatures, they themselves contain ```,``` in their values. We separate values with a special delimiter ```$*$``` to make sure it is not contained any values in a row. This delimiter is defined [here](https://github.com/nimakarimipour/NullAway/blob/031bae163089e43b14107937de2460c8ce4d8664/nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java#L47). With that said, please let me know if you still prefer removing the interface, thank you very much.

Reply:
Drive-by comment: a common alternative to CSV is [TSV](https://en.wikipedia.org/wiki/Tab-separated_values), i.e. tab-separated values. It's often more readable and the separator is very unlikely to clash with the data itself.

Reply:
Hi @Stephan202, thank you very much for the suggestion. ```TSV``` makes a lot more sense. I just applied it. 

With best regards, 

Nima

Reply:
0c3f775b12709e4645771d98b4146c541370c63e

Applied to ```tsv``` and removed the interface.

Reply:
@msridhar I believe this is ready for another review.